### ISSUE TITLE
Social event creation causes 500 error.

### ISSUE THREAD

**Issue by sidneyjennings** [csyager/greeklink-core#75]
If a social event is created without entering a valid date or time, a validation error occurs causing a 500.

**Comment by jonahanderson** [comment]
We might be able to fix this with form errors catching it before creating the event

**Comment by csyager** [comment]
Yeah we can write a validation function on the form.  Theres probably a bunch of these things floating around
